backend/program/indexers/trakt.py

```diff
@@ -65,59 +73,55 @@ def should_submit(item: MediaItem) -> bool:
             interval = timedelta(seconds=settings.update_interval)
             return datetime.now() - item.indexed_at > interval
         except Exception:
-            logger.error(f"Failed to parse date: {item.indexed_at} with format: {interval}")
+            logger.error(f"Failed to parse date: {item.indexed_at}")
             return False
 
     @staticmethod
     def _add_seasons_to_show(show: Show, imdb_id: str):
         """Add seasons to the given show using Trakt API."""
-        if not isinstance(show, Show):

```

Comment:
why get rid of validation?

Reply:
Cause I'm dumb. I just added it back in.